contrib/kubernetes/manifests/registry-s3-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-keyfile
+type: Opaque
+data:
+  accesskey: {base64 encode of the access key}
+  secretkey: {base64 encode of the secret key}
+  region: {base64 encode of the region name}
+  bucket: {base64 encode of the buket name}
```

Comment:
I think we should come up with a uniform storage configuration standards. Having different storage configurations for the same S3 backend for different applicaitons make things confusing sometimes. I think this would make it easy for HELM to template a separate STORAGE backend for entire deis-dev chart